pandas/plotting/_style.py

```diff
@@ -72,18 +70,9 @@ def _maybe_valid_colors(colors):
         # check whether each character can be convertible to colors
         maybe_color_cycle = _maybe_valid_colors(list(colors))
         if maybe_single_color and maybe_color_cycle and len(colors) > 1:
-            # Special case for single str 'CN' match and convert to hex
-            # for supporting matplotlib < 2.0.0
-            if re.match(r'\AC[0-9]\Z', colors) and _mpl_ge_2_0_0():
```

Comment:
Think we still need the condition to check the pattern match here, no?

Reply:
It already said under `else` that "this may no longer be required". So I wanted to see if removing it still passes the test suite